.github/workflows/ci.yml

```
@@ -26,27 +26,121 @@
         else
-          go test -v -race -coverprofile=coverage.out ./...
+          # macOS: Run tests sequentially with coverage to avoid race condition issues
+          go test -v -p 1 -timeout=15m -coverprofile=coverage-${{ matrix.os }}.out ./...
         fi
     
     - name: Upload coverage to Codecov
       if: matrix.os == 'ubuntu-latest' && matrix.go-version == '1.23'
       uses: codecov/codecov-action@v5
       with:
-        file: ./coverage.out
+        file: ./coverage-${{ matrix.os }}.out
```

actionlint (1.7.7)

input "file" is not defined in action "codecov/codecov-action@v5". available inputs are "binary", "codecov_yml_path", "commit_parent", "directory", "disable_file_fixes", "disable_safe_directory", "disable_search", "dry_run", "env_vars", "exclude", "fail_ci_if_error", "files", "flags", "gcov_args", "gcov_executable", "gcov_ignore", "gcov_include", "git_service", "handle_no_reports_found", "job_code", "name", "network_filter", "network_prefix", "os", "override_branch", "override_build", "override_build_url", "override_commit", "override_pr", "plugins", "report_code", "report_type", "root_dir", "skip_validation", "slug", "swift_project", "token", "url", "use_legacy_upload_endpoint", "use_oidc", "verbose", "version", "working-directory"

(action)